### PR TITLE
Chore/update trezor common

### PR DESCRIPTION
**Pull request by szymonlesisz**
## Description

Update trezor-common submodule and build new coins.json

- updated DOGE dust_limit
- removed multiple fee levels for BCH, BTG, DGB
- removed MUE (MonetaryUnit) coin
- removed dead blockbooks (blockchain_link)

related to https://github.com/trezor/trezor-suite/pull/9176

### CODE REVIEW

File: packages/connect-common/files/coins.json [hunk 1]
```diff
@@ -2226,7 +2155,23 @@
         }
     ],
     "erc20": [],
-    "eth": [],
+    "eth": [
```
Comment (mroz22) [inline packages/connect-common/files/coins.json]: just please manually remove this eth record. this is of course temporary solution, automation script should be improved, I am sure I will get to it one day.

**Review by mroz22 — APPROVED** [review]
just one humble request, otherwise 👍

File: packages/connect-common/files/coins.json (hunk 1 above)
Comment (szymonlesisz) [reply packages/connect-common/files/coins.json]: ah, ok i did it for you by excluding them completely using -E param

**Comment by szymonlesisz** [comment]
forcepushed and fixed exclusion of "eth" and "erc20" from coins.json. ~~Their absence should not break anything~~ it did so i fixed it in DataManager [1795d4e](https://github.com/trezor/trezor-suite/pull/9188/commits/1795d4ebb901c99129268d2b3cc6113ecd27764a)

**Review by mroz22 — APPROVED** [review]
thanks!

**Pull request merged**